src/main/java/net/dv8tion/jda/api/entities/Role.java

```diff
@@ -97,6 +97,20 @@
      */
     boolean isMentionable();
 
+    /**
+     * Whether or not this Role is the server's Server Booster role.
+     *
+     * @ return True, if Role is Server Booster role.
+     */
+    boolean isBoosterRole();
```

Comment:
This is probably not necessary anymore, since there is a getRoleType method now.
It would be nice to add more details to both the getRoleType and the isManaged() javadocs

Reply:
Removed, didn't do the javadocs yet